api/main.py

```diff
@@ -1,11 +1,51 @@
-from flask import Flask, request, jsonify
+from flask import Flask, request, jsonify, render_template
+import subprocess
+import os
 app = Flask(__name__)
 
+APP_ROOT = os.path.dirname(os.path.abspath(__file__))
+
+#Homepage of the webapp
+#Creates a folder known_people
+@app.route('/')
+def index():
+    return render_template('index.html')
+
+#Upload Part
+#Uploaded Images are saved in the images folder
+@app.route('/upload', methods=['POST'])
+def upload():
+    target = os.path.join(APP_ROOT, 'images/')
+    print(target)
+
+    if not os.path.isdir(target):
+        os.makedirs(target)
+
+    for file in request.files.getlist('file'):
+        print(file)
+        filename = file.filename
+        destination = "/".join([target, filename])
+        print(destination)
+        file.save(destination)
+
+    count = 1
+    path = os.getcwd()+'/images/'
+    for filename in os.listdir(path):
+        ext = filename[filename.index('.'):]
+        newname = str(count)+ ext
+        os.rename(path+filename,path+newname)
+        count+=1
+
+
+    return render_template('upload_completed.html')
+
+
 # TODO: Complete this method
 # Input: Uploaded image as request body parameter
 # Output: List of people in the image
 @app.route('/peep', methods=["GET"])
 def peep():
+    subprocess.call(['face_recognition', './known_people/', './images/'],stdout=open('Output.txt', 'wb'))
```

Comment:
Can you add this `face_recognition` code into the codebase too?

Reply:
Updated the requirements.txt